onnx/optimizer/optimize.cc

```diff
@@ -3,7 +3,8 @@
 
 #include "onnx/optimizer/optimize.h"
 
-namespace ONNX_NAMESPACE { namespace optimization {
+namespace ONNX_NAMESPACE {
+namespace optimization {
 
 // TODO: Remove this static reference
 static Optimizer _optimizer;
```

Comment:
We can have a static method to return the _optimizer, make sure _optimizer is already initialized.